Makefile

```diff
@@ -148,13 +148,15 @@ install-deps:
 	pip install --upgrade -r requirements.txt
 	python utils/install_actor_deps.py --actor=$(ACTOR)
 
-test:	lint
+test_no_lint:
 	. tut/bin/activate; \
 	python utils/run_pytest.py --actor=$(ACTOR) --report=$(REPORT)
 
+test: lint test_no_lint
```

Comment:
That's really weird but 
`lint test_no_lint` gives me only linter output while `test_no_lint lint` runs both the unittests for actor and the linter for everything as expected.

Reply:
While on travis it works o.O

Reply:
Works for me on rhel7 in VM...

Reply:
If it's not a big deal, can we have `test_no_lint lint` order? )

Reply:
Then you lint after the tests?

Reply:
Is there a problem? To receive +1 from actual ci we would still need to have a happy linter and 100% passing tests regardless of the execution order

Reply:
I just didn't want to change the current behaviour

But instead of a 'it doesn't work for me, so change the order' it would be a good thing to know, __why__ it doesn't work for you